### PR TITLE
post context properties to kodahash/render

### LINKED ISSUE #5: Better message validation
## Context

Currently the `postMessageKoda` is good enough but there are some edge cases that could be potentially fixed by having a coorrect visuals or something

## Task

In `postMessageKoda` replace:
1. id -> to seed
2. in message payload set defined variables that should be consistent (bgcolor, stroke?)
3. add version in the payload

## Ref 

- https://github.com/kodadot/nft-gallery/issues/10310

### PR DESCRIPTION
**Pull request by daiagi**
- Closes #5

### CODE REVIEW

**Comment by daiagi** [comment]
@vikiival 
i can't assign you as reviewer

File: public/sketch.js [hunk 1]
```diff
@@ -49,18 +49,44 @@ function hashToSeed(hash) {
...
+
+  return { details, base64Details };
+}
+
 function postMessageKoda() {
   console.log('Talking')
   const canvas = document.querySelector('canvas')
+const { details, base64Details } = extractCanvasDetails(canvas);
   const message = {
     id: Date.now(),
```
Comment (vikiival) [inline public/sketch.js]: Id should be seed

Comment (daiagi) [reply public/sketch.js]: done

**Comment by vikiival** [comment]
@daiagi have you tested that on art that hasn't been consistent?

**Comment by daiagi** [comment]
i have

* Before




* After

**Pull request merged**